File: src/main/java/com/github/konbluesky/jdph/services/MyProjectService.java
package com.github.konbluesky.jdph.services;

import com.github.konbluesky.javadesignpatternhelperideaplugin.MyBundle;
import com.intellij.openapi.project.Project;

public class MyProjectService {

    private Project project;

    public MyProjectService(Project project) {
        this.project = project;
        System.out.println(MyBundle.message("projectService", project.getName()));
    }
}

File: src/main/kotlin/com/github/konbluesky/javadesignpatternhelperideaplugin/services/MyProjectService.kt
package com.github.konbluesky.javadesignpatternhelperideaplugin.services

import com.github.konbluesky.javadesignpatternhelperideaplugin.MyBundle
import com.intellij.openapi.project.Project

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
